pc2_bits.yml

```diff
@@ -0,0 +1,89 @@
+---
+- name: configure pc2 machines
+  hosts: pc2*,ccsadmin*
```

Comment:
How are the DOMjudge machines called? Can the pc2-ccsadmin machines get pc2 in the name?

Reply:
in the future.